assets/csidriveroperators/shared-resource/hypershift/mgmt/09_deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shared-resource-csi-driver-operator
+  namespace: ${CONTROLPLANE_NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: shared-resource-csi-driver-operator
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        name: shared-resource-csi-driver-operator
+    spec:
+      containers:
+      - args:
+        - start
+        - --kubeconfig=/etc/guest-kubeconfig/kubeconfig
```

Comment:
There must be `--namespace=openshift-cluster-csi-drivers`, because the operator autodetects `clusters-<hosted cluster name>` in the mgmt cluster, and we need a namespace in the guest cluster.

Reply:
add of namespace arg pushed